experimental/rule_inference/rule_application.py

```diff
@@ -0,0 +1,70 @@
+import attr
+import toml
+from typing import List
+from polyglot_piranha import (
+    Rule,
+    PiranhaArguments,
+    RuleGraph,
+    execute_piranha,
+    PiranhaOutputSummary,
+)
+
+
+@attr.s
+class CodebaseRefactorer:
```

Comment:
I'm not sure we need this at all for now. But maybe at some point we want to add more functionality rather than just applying the rules